documentation/asciidoc/accessories/display/legacy.adoc

```diff
@@ -1,6 +1,31 @@
 == Legacy Support
 
-NOTE: These instructions are for the original Raspberry Pi 1 Model A and B boards only. You can identify an original board as it is the only model with a 26-pin GPIO header, all other models have the now-standard 40-pin connector.
+[NOTE]
+====
+Older Raspberry Pi hardware, and older versions of Raspberry Pi OS, with X11 can install an on-screen keyboard and emulate right clicks on the touchscreen. 
```

Comment:
IMHO this is a bit too vague - what exactly counts as "older hardware" or "older software"?
Perhaps "If you are running a version of Raspberry Pi OS prior to Bookworm, or if you are running Raspberry Pi OS Bookworm on a model of Raspberry Pi older than a Raspberry Pi 4, then..."? Although that's a bit too wordy, so perhaps "Older Raspberry Pi hardware (before Raspberry Pi 4), and older versions of Raspberry Pi OS (before Bookworm), ..."? :man_shrugging:

Reply:
Used "pre-" instead of "before", but it's a good callout regardless.